DrawerController/DrawerController.swift

```diff
@@ -225,10 +225,32 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
     }
     
     /// Actions
+    @IBAction func openLeftSide(_ sender: Any) {
+        self.openSide(.left)
+    }
+    
+    @IBAction func openRightSide(_ sender: Any) {
```

Comment:
Can it be used externally even if it is not specified as public?

Reply:
Yes. `@IBAction` seems to make it public.